README.md

```diff
@@ -0,0 +1,67 @@
+  at System.Linq.Enumerable.Single[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate) <0x2146c98 + 0x000f0> in <filename unknown>:0 
+  at System.Linq.Dynamic.Core.DynamicQueryableExtensions.GetMethod (System.String name, System.Int32 parameterCount, System.Func`2[T,TResult] predicate) <0x2146898 + 0x0004c> in <filename unknown>:0 
+   --- End of inner exception stack trace ---
+  at System.Linq.Dynamic.Core.DynamicQueryableExtensions.GetMethod (System.String name, System.Int32 parameterCount, System.Func`2[T,TResult] predicate) <0x2146898 + 0x00078> in <filename unknown>:0 
+  at System.Linq.Dynamic.Core.DynamicQueryableExtensions..cctor () <0x213ca70 + 0x00024> in <filename unknown>:0 
+   --- End of inner exception stack trace ---
+  at ComponentLibWithXmlLinkerConfig.Component1.OnClick () [0x00001] in C:\users\user\Documents\GitHub\danroth27\ComponentLibWithXmlLinkerConfig\ComponentLibWithXmlLinkerConfig\Component1.razor:33 
+  at (wrapper delegate-invoke) <Module>.invoke_void()
+  at Microsoft.AspNetCore.Components.EventCallbackWorkItem.InvokeAsync[T] (System.MulticastDelegate delegate, T arg) <0x20d8fd0 + 0x00062> in <filename unknown>:0 
+  at Microsoft.AspNetCore.Components.EventCallbackWorkItem.InvokeAsync (System.Object arg) <0x20d8f10 + 0x0000a> in <filename unknown>:0 
+  at Microsoft.AspNetCore.Components.ComponentBase.Microsoft.AspNetCore.Components.IHandleEvent.HandleEventAsync (Microsoft.AspNetCore.Components.EventCallbackWorkItem callback, System.Object arg) <0x20d8e78 + 0x0000a> in <filename unknown>:0 
+  at Microsoft.AspNetCore.Components.EventCallback.InvokeAsync (System.Object arg) <0x20d8a00 + 0x00040> in <filename unknown>:0 
+  at Microsoft.AspNetCore.Components.RenderTree.Renderer.DispatchEventAsync (System.UInt64 eventHandlerId, Microsoft.AspNetCore.Components.RenderTree.EventFieldInfo fieldInfo, System.EventArgs eventArgs) <0x20c9058 + 0x000a8> in <filename unknown>:0 
+```
+
+To preserve the `System.Linq.Queryable` type and its members, the following XML linker config file is added to the ComponentLibWithXmlLinkerConfig project:
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- IL linker format: https://github.com/mono/linker#link-xml-file-examples -->
+<linker>
+  <assembly fullname="System.Core">
+    <type fullname="System.Linq.Queryable" preserve="all" />
+  </assembly>
+</linker>
+```
+
+This file is then embedded into the library by adding the following to the project file:
+
+```xml
+<ItemGroup>
+  <EmbeddedResource Include="ComponentLibWithXmlLinkerConfig.xml">
+    <LogicalName>$(MSBuildProjectName).xml</LogicalName>
+  </EmbeddedResource>
+</ItemGroup>
+```
+
+> Note that the name of the embedded XML linker config file must match the name of the assembly. By default the file name will be prefixed with the assembly namespace, so the correct name must be explicitly specified as the `LogiclName` for the embedded file.
```

Comment:
Typo in LogiclName

Also it would be a bit clearer if the physical file name was *not* the same as the project name, otherwise people might misunderstand and think it has to be. I suggest simply calling it LinkerConfig.xml.

Reply:
Thanks! Fixed.